returns/future.py

```diff
@@ -437,6 +437,31 @@ def decorator(*args, **kwargs):
     return decorator
 
 
+def asyncize(function: Callable[..., _ValueType]) -> Callable[
+    ...,
+    Coroutine[Any, Any, _ValueType],
```

Comment:
Can you please add a note here that it is useful for composition with `Future` and `FutureResult`?

Reply:
Done!